src/sonic-yang-models/doc/Configuration.md

```diff
@@ -1,2768 +1,2770 @@
-        }
-        "synchronous_mode": {
-            "status": "enable"
-        }
-        "dhcp_server": {
-            "status": "enable"
-        }
-    }
-```
-The default value of flags in `SYSTEM_DEFAULTS` table can be set in `init_cfg.json` and loaded into db at system startup. These flags are usually set at image being build, and are unlikely to change at runtime.
-
-If the values in `config_db.json` is changed by user, it will not be rewritten back by `init_cfg.json` as `config_db.json` is loaded after `init_cfg.json` in [docker_image_ctl.j2](https://github.com/Azure/sonic-buildimage/blob/master/files/build_templates/docker_image_ctl.j2)
-
-For the flags that can be changed by reconfiguration, we can update entries in `minigraph.xml`, and parse the new values in to config_db with minigraph parser at reloading minigraph. If there are duplicated entries in `init_cfg.json` and `minigraph.xml`, the values in `minigraph.xml` will overwritten the values defined in `init_cfg.json`.
-
-### RADIUS
-
-The RADIUS and RADIUS_SERVER tables define RADIUS configuration parameters. RADIUS table carries global configuration while RADIUS_SERVER table carries per server configuration.
-
-```
-   "RADIUS": {
-       "global": {
-              "auth_type": "pap",
-              "timeout": "5"
-        }
-    }
-
-    "RADIUS_SERVER": {
-        "192.168.1.2": {
-               "priority": "4",
-               "retransmit": "2",
-               "timeout": "5"
-        }
-    }
-```
-
-### Static DNS
-
-The DNS_NAMESERVER table introduces static DNS nameservers configuration.
-
-```json
-{
-	"DNS_NAMESERVER": {
-		"1.1.1.1": {},
-		"fe80:1000:2000:3000::1": {}
-	},
-}
-```
-
-### FIPS
-
-The FIPS table introduces FIPS  configuration.
-
-```json
-{
-    "FIPS": {
-        "global" : {
-            "enable": "true",
-            "enforce": "false"
-        }
-    }
-}
-```
-### MID_PLANE_BRIDGE"
-
-The MID_PLANE_BRIDGE" table introduces the configuration for the midplane bridge interface for Smart Switch.
-
-```json
-{
-    "MID_PLANE_BRIDGE": {
-        "GLOBAL" : {
-            "bridge": "bridge_midplane",
-            "ip_prefix": "169.254.200.254/24"
-        }
-    }
-}
-```
-
-### DPUS
-
-The DPUS table introduces the information on the DPUs (Data Processing Unit) available on the platform.
-
-```json
-{
-    "DPUS": {
-        "dpu0": {
-            "midplane_interface": "dpu0"
-        },
-        "dpu1": {
-            "midplane_interface": "dpu1"
-        }
-    }
-}
-```
-
-#### 5.2.3 Update value directly in db memory
-
-For Developers
-==============
-
-Generating Application Config by Jinja2 Template
-------------------------------------------------
-
-To be added.
-
-Incremental Configuration by Subscribing to ConfigDB
-----------------------------------------------------
-
-Detail instruction to be added. A sample could be found in this
-[PR](https://github.com/Azure/sonic-buildimage/pull/861) that
-implemented dynamic configuration for BGP.
+
```

Comment:
you have changed the entire file, that's not acceptable

Reply:
Ok, let me fix this

Reply:
Fixed.